### DISCUSSION TITLE
blog/2023/03/career-retrospective-2022-late/

### DISCUSSION THREAD

**Discussion by giscus** [d0lim/blog-comments#12] [Announcements]
# blog/2023/03/career-retrospective-2022-late/

2022년 하반기 취준 회고입니다. 주절주절 주의

https://d0lim.com/blog/2023/03/career-retrospective-2022-late/

**Comment by Beopsik** [comment]
잘 봤습니다~~ MSA부분 공감되네요. 거대한 MSA 서비스에서 일부분을 담당하는 사람은 MSA인가 인지를 못하겠더라고요 ㅋㅋㅋ OOP와 디자인 패턴을 공부해서 자기가 맡은 일부분의 서비스를 기깔나게 만드는 사람이 되는게 낫다고 생각합니다

**Reply by dkile** [reply]
인정합니다

**Comment by dkile** [comment]
잘 보고 갑니다~

**Comment by redjen8** [comment]
자바는 죄가 없다.. 자바 사용자를 석방하라.!

**Reply by Beopsik** [reply]
그러면,,,스칼라 혁명단에 가입하시겠습니까? 우리는 형제 아닙니까,,
